### DISCUSSION TITLE
Change output order

### DISCUSSION THREAD

**Discussion by gassganso** [belowaverage-org/phonebook#28] [Q&A] [answered]
Hi,
A very handful job done by developers here. I've been looking here and there trying to make the results output ordered by "description".
Any hint that can help me?
Thanks.

**Comment by krisdb2009** [comment]
That shouldn't be a problem.

Let me release a new version that allows the sorting be customized.

**Comment by krisdb2009** [comment]
#29

**Comment by krisdb2009** [comment, marked as answer]
This feature has been added. And the documentation has been updated to reflect these changes:

https://wiki.belowaverage.org/software/phonebook#modifying-the-schema

The gist of it is you need to move the "orderby" attribute to the column of your choice to change how the front-end retrieves the order of numbers.

![image](https://user-images.githubusercontent.com/8226974/176045164-5f2a543b-463a-491c-9a45-cf9914282632.png)

I should also note that when you change this file you need to clear your browser cache. The caching is very aggressive and you may need to go into the developer tools in order to clear the cache. I recommend testing your changes by opening a fresh instance of incognito mode whenever you make a change.

**Reply by gassganso** [reply]
It works like a charm. As I said before: "A very handful job done by developers here".
Thank you very much.
